docs/getting-started/continuous_integration.md

```diff
@@ -98,61 +98,41 @@ not set this.
 #### Adding Other Sanitizers
 CIFuzz supports address, memory and undefined sanitizers. Address is the default
 sanitizer and will be used for every job in which a sanitizer is not specified.
-To add another sanitizer to your workflow copy the `Fuzzing` job and rename it
-to the sanitizer you want to fuzz with. Then add the sanitizer variable to both
-the `Build Fuzzers` step and the `Run Fuzzers` step. The choices are `'address'`,
-`'memory'`, and `'undefined'`. Once this additional job is configured the CIFuzz
-workflow will run all of the jobs corresponding to each sanitizer simultaneously.
+To specify a sanitizer explicitly add the sanitizer variable to both the `Build Fuzzers`
+step and the `Run Fuzzers` step. The choices are `'address'`, `'memory'`, and `'undefined'`.
```

Comment:
I think the comment here is slightly confusing because of the matrix  we use below. Maybe say: `To use a sanitizer add it to the list of sanitizers in the matrix field below. The choices are ...`

Reply:
I was trying to document the "sanitizer" variable itself (by analogy with the other variables) and then give an example of how it can be used. I agree that in its current form it seems to be confusing but I'm not sure what the best wording would be.

Reply:
I wonder if it would help if I gave a link to https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix somewhere to make it clear that the "sanitizer" there is different from the "sanitizer" variable (which I think could be documented in a separate section)?

Reply:
How about we document the sanitizer variable and then mention because we are using the matrix, put the sanitizers you want in the list? ie: your wording followed by mine with a link to the matrix documentation?

Reply:
Updated. Hope it's better.

Reply:
@jonathanmetzman I wonder what's used to render https://google.github.io/oss-fuzz/getting-started/continuous-integration/. It seems I should have escaped dollar signs somehow:
![Screenshot 2020-06-16 at 18 37 42](https://user-images.githubusercontent.com/10195800/84796022-dc41a680-b000-11ea-86b8-061712522f1a.png)

Reply:
I've just opened https://github.com/google/oss-fuzz/pull/3994